packages/core/src/textures/resources/CubeResource.ts

```diff
@@ -15,7 +18,9 @@ import { TARGETS } from '@pixi/constants';
  */
 export class CubeResource extends ArrayResource
 {
-    constructor(source, options)
+    items: Array<BaseTexture>;
+
+    constructor(source: Array<string|Resource>, options?: any)
```

Comment:
The options here are not any

Reply:
they are being passed to autoDetectResource, we only can do something like `{[key: string: any}`

Reply:
You need stricter types here for options. My suggestion is to make CubeResourceOptions and then combine all the options into the autoDetectResource: CubeResourceOptions|CanvasResourceOptions|VideoResourseOptions

Reply:
You need stricter types here for options. My suggestion is to make CubeResourceOptions and then combine all the options into the autoDetectResource: CubeResourceOptions|CanvasResourceOptions|VideoResourseOptions

Reply:
^^^ its up to you. However I remind you that "|" might have side effects. I remember our struggle with `Point|ObservablePoint` ;)

Reply:
Yeah that’s fine in this case because these Resource-specific options are non-functional and don’t have method signatures like Point v ObservablePoint.